woocommerce/pinterest-for-woocommerce
# Add onboarding eligibility notice

### Changes proposed in this Pull Request:

Closes #515.

Implement eligibility notice on connect account step as described in the issue description. As described in the issue, the link is a placeholder for now.

### Screenshots:

![image](https://user-images.githubusercontent.com/40774170/187291398-8aba8a90-7dbf-4fba-946a-16b07b894c9a.png)

### Detailed test instructions:
1. Install the plugin.
2. Go to the first step of the onboarding.
3. The eligibility notice should be displayed.
4. The event should be tracked when the link is clicked.


### Additional details:

### Changelog entry

> Add - Onboarding eligibility notice for ad credits
